google-cloud-spanner/src/main/java/com/google/cloud/spanner/Options.java

```diff
@@ -141,7 +141,7 @@ public static ReadQueryUpdateTransactionOption priority(RpcPriority priority) {
     return new PriorityOption(priority);
   }
 
-  public static ReadQueryUpdateTransactionOption maxCommitDelay(Duration maxCommitDelay) {
+  public static TransactionOption maxCommitDelay(Duration maxCommitDelay) {
```

Comment:
nit: Can we also add a short javadoc comment to this public method that indicates where/how to use it?

Reply:
I have asked Navarre from the backend team to add this since I didn't have the exact details on how our external docs are explaining this feature.